get-started/index.md

```diff
@@ -41,12 +41,16 @@ $ kubectl get pods -n olm
   summary="Cryostat on OperatorHub"
   image-name="cryostat-operatorhub-search.png"
 %}
-Use the search bar to find the Cryostat (provided by Red Hat) catalog item.
+Use the search bar to find the **Red Hat build of Cryostat** catalog item.
```

Comment:
We need to update this screenshot since it has a provider as `Catalog Under Test`? It should be `Red Hat`.

Reply:
Oh, haha. I faked these screenshots by installing the upstream Operator build and then modifying the page DOM to fix texts, but I missed that one.

Are you able to get that replacement updated screenshot? It might be a few days before I have time to get around to setting everything up to be able to get it.

Reply:
Yeh how about this?

![image](https://github.com/cryostatio/cryostatio.github.io/assets/68053619/3c36f584-cc00-4535-b4ab-15525b094d42)